src/config/index.ts

```diff
@@ -29,3 +29,5 @@ export const LOTTERY_TICKET_PRICE = 1
 export const DEFAULT_TOKEN_DECIMAL = BIG_TEN.pow(18)
 export const DEFAULT_GAS_LIMIT = 200000
 export const DEFAULT_GAS_PRICE = 5
+export const TESTNET_CHAIN_ID = '97'
```

Comment:
remove and uses `@pancakeswap-libs/sdk`

Reply:
👍 Done here https://github.com/pancakeswap/pancake-frontend/pull/1554